src/cmd-buildextend-metal

```diff
@@ -172,7 +175,13 @@ if [ -z "${use_anaconda}" ]; then
     if [ -n "${ref_is_temp}" ]; then
         ref_arg=${commit}
     fi
-    runvm -drive "if=virtio,id=target,format=${image_format},file=${path}.tmp" -- \
+    target_drive="if=virtio,id=target,format=${image_format},file=${path}.tmp"
+    target_device=""
+    if [[ $image_format == raw && $image_type == dasd ]]; then
+        target_drive="if=none,id=target,format=${image_format},file=${path}.tmp"
+        target_device="-device virtio-blk-ccw,devno=fe.0.0004,drive=target,physical_block_size=4096,logical_block_size=4096,scsi=off"
+    fi
+    runvm -drive "$target_drive" $target_device -- \
```

Comment:
`$target_device` without quotes is on purpose.

Reply:
You're looking for:

```
    # shellcheck disable=SC2086
```

:)

Reply:
I removed the wrapper quotes for `$target_device` because when parsed to qemu command, it would be like:
`qemu '-device virtio-blk-ccw,...'`

but the correct one should be :
`qemu -device 'virtio-blk-ccw,...'`

I think I'm going to make an array:
`target_device=( "-device" "virtio-blk-ccw,devno=fe.0.0004,drive=target,physical_block_size=4096,logical_block_size=4096,scsi=off" )`

Reply:
Hah, which is pretty much is the same on `cmdlib.sh` before it was moved here recently.